MAUI/Scheduler/migration.md

```diff
@@ -975,6 +975,12 @@ this.Content = scheduler;
 <td>{{'[ResourceIds]()'| markdownify }}</td>
 <td>Gets or sets the resource ids property for the schedule appointment to allocate the corresponding resource to the appointment.</td></tr>
 
+<tr>    
+<td>{{'[TextColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.ScheduleAppointment.html#Syncfusion_SfSchedule_XForms_ScheduleAppointment_TextColor)'| markdownify }}</td>
```

Comment:
update TextColormapping in all examples
https://help.syncfusion.com/maui/scheduler/appointments#creating-business-objects